joblib/test/test_parallel.py

```diff
@@ -440,6 +444,35 @@ def test_multiple_spawning():
                   [delayed(_reload_joblib)() for i in range(10)])
 
 
+class TestParallelBackend(SequentialBackend):
```

Comment:
I would call this class "MyParallelBackend", the reason is that a class starting with "Test..." can be interpreted as a unit test class, discovered by nose.